app/posts/manage-teacher-training-applications/2020-05-18-access-control-history.md

```diff
@@ -0,0 +1,551 @@
+
+[Organisational diagrams folder](https://drive.google.com/drive/u/0/folders/1p4Ap6BwUsEi4X_5xuqY1wp2QgY6IUHf5)
+
+#### Safeguarding research
+
+During research, we identified a candidate need to explain any criminal convictions:
+
+>__As a__ candidate
+>__I need to__ feel comfortable disclosing negative information about my past
+>__So that__ I know whether it’s worth carrying on with my application
+
+[The questions we ask candidates](https://apply-beta-prototype.herokuapp.com/application/undefined/suitability/)
+
+In completing this section, candidates may supply sensitive data. We tested this information with providers who expressed a strong interest in seeing this data. They also highlighted that the data would need to be locked down to certain individuals. This will be the same case as the HESA data, which providers need to report into HEI systems or DTTP.
+
+For SCITTs this may be:
+* Directors, chairs, CEOs
+* HR 
+* Possibly admin staff
+
+For HEIs this may be: 
+* ITT lead/director
+* Criminal convictions teams
+* HR teams
+
+We made a decision, as an MVP, to collect the data from candidates, but only show a simple “candidate has shared information” or “candidate has not shared information”. We then steer providers to manage the communication about the information offline.
+
+However, this does not support the need from providers to see the sensitive data. There is an opportunity to build this safeguarding information (HESA and criminal convictions) into the access management within Manage. We realised we could do this in such a way that it would be the provider’s responsibility to give permission to the relevant people within their organisation to view this information.
+
+#### In summary
+
+The research shows that:
+* Providers organise themselves in a variety of ways. We have identified a number of common organisational structures, some more straightforward than others. We learnt it is possible for providers to exist within a highly interconnected network of organisations.
+* Providers can have different ways of working with one another. A single provider, typically a School Direct, can have multiple different ways of working depending upon the accredited body which ratifies a particular  course for a particular application.
+* There are multiple roles within providers, each of which may require different access permissions. For example:
+    * An ITT Director may be the only person who should view safeguarding information
+    * A subject lead at a school may only need to view applications
+    * An ITT Administrator may need to make decisions and manage who should have access to the system.
+
+### User needs
+
+#### Sharing applications with other users
+
+##### Key needs
+>__As an__ administrator within a provider
+>__I need to__ share an application with other decision-makers (potentially in other organisations)
+>__So that__ we can decide how to progress the application
+
+##### Supporting needs
```

Comment:
Again, we’re getting pretty deeply nested here. If we do decide we need 5 heading levels, we should review what a level 5 heading looks like; right now it renders as light grey text.